php-checks/src/main/resources/org/sonar/l10n/php/rules/php/Drupal_profile.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Drupal",
+  "ruleKeys": [
+    "S101",
+    "S103",
+    "S105",
+    "S113",
+    "S115",
+    "S116",
+    "S117",
+    "S121",
+    "S1105",
+    "S1109",
+    "S1131",
+    "S1757",
+    "S1765",
+    "S1779",
+    "S1780",
+    "S1784",
+    "S1788",
+    "S1793",
+    "S1808",
+    "S2003",
+    "S4833"
+  ]
+}
```

Comment:
We should not create this new file. We already have [drupal-profile.xml](https://github.com/SonarSource/sonar-php/blob/master/sonar-php-plugin/src/main/resources/org/sonar/plugins/php/profile/drupal-profile.xml) which contains a more detailed configuration of the profile.

Same thing for PSR-2_profile.json.

Reply:
Ok, I remove them both and create a new PR?